rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-08-13
```

Comment:
Why is this project using nightly? Can we not compile on stable?

Reply:
Because of the secp25kfun crate:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/froyer/.cargo/git/checkouts/secp256kfun-fb74af16957b2754/cdfbc76/secp256kfun/src/lib.rs:1:1
  |
1 | #![feature(rustc_attrs, min_specialization, external_doc)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Oh didn't know we were using secpfun.